scripts/cleanup_ebpf_cicd_tests.ps1

```diff
@@ -2,6 +2,7 @@
 # SPDX-License-Identifier: MIT
 
 param ([parameter(Mandatory=$false)][string] $Target = "TEST_VM",
+       [parameter(Mandatory=$true)][bool] $KmTracing,
```

Comment:
Just wondering - is there a reason why this is a required parameter, instead of having a default value that we fall back to?

Reply:
I've mentioned the reasoning in a prior [response](https://github.com/microsoft/ebpf-for-windows/pull/2946#discussion_r1360859710)